e2e/upgrade-testing-rbd.go

```diff
@@ -0,0 +1,202 @@
+package e2e
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo" // nolint
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
+)
+
+var (
+
+	// cwd stores the initial working directory.
+	cwd string
+
+	// subvolumegroup contains the subvolume group name
+	// used for e2e testing.
+	subvolumegroup string
+)
+
+var _ = Describe("RBD Upgrade Testing", func() {
+	f := framework.NewDefaultFramework("upgrade-test-rbd")
+	var (
+		c       clientset.Interface
+		pvc     *v1.PersistentVolumeClaim
+		app     *v1.Pod
+		err     error
```

Comment:
remove err and pvcSize from here

Reply:
I'm a bit confused... How can we get the 1gb size from the spec?

Reply:
pvc.Spec.Resources.Requests[v1.ResourceStorage] you need to extract it add a todo to do enhancement let's get this in first